### ISSUE TITLE
Increase PSQ Timeout

### ISSUE THREAD

**Issue by aarontp** [google/turbinia#215]
The timeout for PSQ workers pulling requests off the queue seems very low (10 seconds maybe?).  We should find a way to increase this to something larger.

**Comment by fpiedrah** [comment]
@aarontp I would like to take this task too! 

I've been watching some PSQ examples, this one in particular seems to be helpful: https://github.com/GoogleCloudPlatform/psq/blob/master/examples/run.py

This shows that we could increase the timeout by passing a timeout parameter on the queues `.result()` method. I'm thinking on adding this on `PSQTaskManager.process_tasks()` in line 415 we have `task.result = task.stub.result()` we can add the `timeout` tho that result, but it should be parametrized somewhere.

What do you think about this?

**Comment by aarontp** [comment]
Sounds good;  What is the timeout set to by default?  For parameterization I think we can just add something like a TASK_TIMEOUT somewhere near the top of task_manager.py.  I don't think it needs to be a config var.

**Comment by fpiedrah** [comment]
@aarontp seems like the default value is None, meaning that the task will wait forever to complete.

https://github.com/GoogleCloudPlatform/psq/blob/e27e2f37520f1f0d3006b8ba8ea214e8a57c7563/psq/task.py#L148

**Comment by aarontp** [comment]
Huh, interesting.  Thanks for tracking that down.  I wonder if it's always been that way since in the past it seemed to time out, but maybe I was seeing other problems with scheduling that just looked like timeouts.

In that case it's a pretty low priority compared to other things, but it might be nice to have something set anyway just to be explicit, and so that things will eventually error out.  Tasks can potentially take a couple days or more to run, so I suggest setting the timeout at one week for now, and one it's parameterized it will be easy to change later.

**Comment by ericzinnikas** [comment]
Yeah, I've been finagling with something similar on the Celery side. I have some changes TODO that can re-use the timeout you're adding.
